### DISCUSSION TITLE
GET_PLOTS to be avaliable on harvester by a multi farm call

### DISCUSSION THREAD

**Discussion by filzek** [Chia-Network/chia-blockchain#6401] [Ideas]
The current design of the command GET_PLOTS relay only inside the current machine, if you are in a multifarm cenario the main farmer cant know exactly the numbers and info from the plots in the system.

It is very important to also add in the 8447 farmer connection list the ability that the farmer post a request in GET_PLOTS so all plots will be know by the farmer on the multi farm.

Please advise about this resource as it is very simple to add and very necessary for the currently Plots system.

**Comment by IgnaciusMaximus** [comment]
You mean that in that scenario the farmer must be available the port 8447? 

What about the harvesters? Any port change?
